CONTRIBUTING.md

```diff
@@ -33,11 +33,12 @@ make install-deps
 # install dev/tests dependencies
 make install-deps-test
 
-# code formatting (run this as a pre-commit step for your code changes)
+# code formatting
+# (this is also run automaticall as part of pre-commit hook)
 make format
 
-# code style and docstring style (run this as a pre-commit step for your
-# code changes)
+# code style and docstring style
+# (this is also run automaticall as part of pre-commit hook)
```

Comment:
```suggestion
# (this is also run automatically as part of the pre-commit hook)
```

same question as above though

Reply:
ah my bad, the "dotfile" was ignored. sorry!!!

Reply:
fixed too @bparees , updated doc how to setup it